.phylum_project

```diff
@@ -0,0 +1,4 @@
+---
+id: 112b5ffb-b78a-4e60-81e8-8ce8265ec154
```

Comment:
Was this `.phylum_project` file generated by the `phylum_bot` account?

Reply:
No. Should it be? I created it in the root of the repo with:
```
phylum projects create phylum-ci
```

Reply:
The project was re-created with the `phylum_bot` account.